include/config_defaults.h

```diff
@@ -220,6 +220,11 @@
 #ifndef TOOL_WATERLEVEL
 #define TOOL_WATERLEVEL 1
 #endif
+#ifndef OSW_EMULATOR
+#define OPENWEATHERMAP_APIKEY ""
+#define OPENWEATHERMAP_CITY "turin"
+#define OPENWEATHERMAP_STATE_CODE "IT"
+#endif
```

Comment:
Please follow the pattern of the other defines and only re-define them is they were not set before (independent of `OSW_EMULATOR`).

Reply:
Also... As your app is under experimentals... Maybe also move it a little bit down?